marklogic-data-hub-central/ui/e2e/cypress/e2e/modeling/verifyConceptClassesInModeling.cy.tsx

```diff
@@ -365,4 +365,44 @@ describe("Concept classes in Modeling screen", () => {
     entityTypeTable.getConceptClass("TestConcept").should("not.exist");
   });
 
+  it("Create/Delete new relationship and new concept class from Table view", {defaultCommandTimeout: 120000}, () => {
```

Comment:
Any reason why we need to change the default timeout?

Reply:
Why not move this it before we delete the TestConcept? That past is already tested in the previous create and edit tests. And focus only on the relationship. Then the delete test cleans the concept.